### PR TITLE
Run flake8 on manager build and configure drone setup

### PR DESCRIPTION
**Pull request by meldafrawi**

### CODE REVIEW

**Comment by yasker** [comment]
This depends on the host flake8 version (if it's installed) instead of version going to be used in the longhorn-test container.

**Comment by yasker** [comment]
Also, we need the ability to trigger drone build for this repo as well.

File: Dockerfile.dapper [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+ENV DAPPER_SOURCE /go/src/github.com/longhorn/longhorn-tests
+
+WORKDIR ${DAPPER_SOURCE}
+
+ENV VIRTUAL_ENV=${DAPPER_SOURCE}/manager/integration/.venv
+
+RUN python2 -m virtualenv --python=/usr/bin/python2 ${VIRTUAL_ENV}
+
+CMD ["./manager/integration/scripts/flake8-check.sh"]
```
Comment (yasker) [inline Dockerfile.dapper]:
```
/usr/bin/docker: Error response from daemon: OCI runtime create failed: container_linux.go:345: starting container process caused "exec: \"./manager/integration/scripts/flake8-check.sh\": permission denied": unknown.
```

Seems flake8-check.sh is not executable.

Comment (meldafrawi) [reply Dockerfile.dapper]: `flake8-check.sh` is now executable

File: manager/integration/scripts/flake8-check.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```
Comment (yasker) [inline manager/integration/scripts/flake8-check.sh]: Trailing space

File: manager/integration/scripts/flake8-check.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash 
+
+source manager/integration/.venv/bin/activate
```
Comment (yasker) [inline manager/integration/scripts/flake8-check.sh]: Trailing space

File: manager/integration/scripts/flake8-check.sh [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash 
+
+source manager/integration/.venv/bin/activate 
+
+pip install -r manager/integration/tests/requirements.txt
+
+flake8 manager/integration/tests
```
Comment (yasker) [inline manager/integration/scripts/flake8-check.sh]: flake8 has no output, add `echo flake8 check succeed` at the end?

Comment (meldafrawi) [reply manager/integration/scripts/flake8-check.sh]: Done

**Pull request merged**